apps/lib/apps.c

```diff
@@ -1805,7 +1805,7 @@ int save_index(const char *dbfile, const char *suffix, CA_DB *db)
 {
     char buf[3][BSIZE];
     BIO *out;
-    int j;
+    size_t j;
```

Comment:
There is a line below for
j = TXT_DB_write(out, db->db); that returns a long..
So that might need a separate variable

Reply:
I'll just revert this and cast the strlens. They cannot really overflow int.